cmd/fyne/internal/commands/package-wasm.go

```diff
@@ -8,22 +8,6 @@ import (
 	"fyne.io/tools/cmd/fyne/internal/templates"
 )
 
-func (p *Packager) packageWeb() error {
```

Comment:
Shouldn't this wasm-only package format be called "web"?
The fact that it's a wasm binary internally should be considered an implementation detail I think.
So let's use the target name web, and use that for the folder as well.

Not sure we have to change this function name, but the file rename and folder here seemed like a good place to comment